### ISSUE TITLE
Remove various old themes

### ISSUE THREAD

**Issue by sinabahram** [pantheon-systems/WordPress#232]
Hello, is it possible to remove the various older themes from this repository, because if one chooses to do so on their own; for example, if you are using your own theme or simply another theme, then the various WP Twenty* themes are still around, bloating the repository. Maybe an option could exist in Pantheon land to use a slim version of this WP instance with no themes provided, or really any other solution as long as it gets them out of there.

**Comment by greg-1-anderson** [comment]
You may remove these themes from your own sites if you wish. Pantheon never modifies the standard WordPress themes that appear in this repo. They are copied in from the first commit that contains them (in a release), and left unmodified thereafter. This is because a lot of Pantheon sites remove or modify these theme files, and if we removed them in the upstream, then we'd cause merge conflicts in all of these sites.

We can't really make a site without any themes, because then you wouldn't be able to install it until you added a theme of your own.

**Comment by sinabahram** [comment]
I understand your response, but please note that this does not reflect my experience.

 

I made a site on Pantheon.

 

I deleted the themes as you suggest.

 

Now I can no longer use Pantheon to update my Wp instance because it complains that I deleted these things. Obviously I deleted them appropriately via git, but maybe I did something wrong here?

 

When  speaking with Pantheon, they also seemed to agree this was an issue, and pointed me to this repository to advocate for this, so maybe I am totally missing something. Happy to be corrected/educated.

**Comment by greg-1-anderson** [comment]
I will have to attempt to reproduce your experience. What version of WordPress did you start with? I presume you are updating to today's most recent version.

**Comment by sinabahram** [comment]
No, this was an earlier 5.X release, but it was absolutely the most current at the time of doing the deletions. I also have an active site on there I’m happy to just go git rm the themes on, and then attempt to do an update to, but I may have to wait for an upstream update to verify of course.

 

Thank you for any findings. Please let me know how I can help.

 

Thanks again

**Comment by greg-1-anderson** [comment]
Do you know what version you were on at the time you attempted to do a dashboard update?

**Comment by sinabahram** [comment]
I want to say 5.1, but I don’t have evidence for that, and I’ve been on the other side of this where people provide nebulous info so I don’t wish to tell you wrong.
